mix.exs

```diff
@@ -24,20 +24,31 @@ defmodule MongoosePush.Mixfile do
 
   defp deps do
     [
-     {:pigeon, git: "https://github.com/rslota/pigeon.git", tag: "6d1e4e3"},
-     {:maru, git: "https://github.com/rslota/maru.git", tag: "7c1da75", override: true},
+     {:pigeon, github: "rslota/pigeon", tag: "087bb38"},
+     {:maru, github: "elixir-maru/maru", tag: "7a24d1a3", override: true},
      {:poison, "~> 3.0"},
-     {:httpoison, "~> 0.10.0"},
+     {:httpoison, "~> 0.11.0"},
      {:maru_swagger, github: "elixir-maru/maru_swagger"},
      {:distillery, "~> 1.0"},
      {:confex, "~> 1.4", override: true},
      {:mix_docker, "~> 0.3"},
+
+     # Just overrides to make elixometer compile...
+     {:setup, github: "uwiger/setup", tag: "1.8.0", override: true, manager: :rebar},
+     {:edown, github: "uwiger/edown", tag: "0.8", override: true},
+     {:lager, ">= 3.2.1", override: true},
+     {:exometer_core, github: "PSPDFKit-labs/exometer_core", override: true},
```

Comment:
Could you explain why `PSPDFKit-labs/exmometer*` are used instead of the original ones form `Feuerlabs`?